lib/config.go

```diff
@@ -80,15 +109,10 @@ func (m *ConfigMethods) GetConfig(ctx context.Context, p *GetConfigParams) ([]by
 	return res, nil
 }
 
-// GetConfigKeys returns the Config key fields, or sub keys of the specified
-// fields of the Config, as a slice of bytes to be used for auto completion
-func (m *ConfigMethods) GetConfigKeys(ctx context.Context, p *GetConfigParams) ([]byte, error) {
-	if m.inst.http != nil {
-		return nil, ErrUnsupportedRPC
```

Comment:
I think we want to keep this behavior for these config methods? We don't want config to be doable over rpc. The check for `http != nil` should be moved into the method definition above, not the impl. I know it breaks the regularity of these method definitions, but I'm think it's the best we can do for now. Maybe the `Mapping` [metadata](https://github.com/qri-io/qri/pull/1687) can determine whether it's allowed to call something over rpc?

Reply:
ahh, yes I misunderstood. I somehow thought this check happened in Dispatch for some reason. Adding back in. And yes, having a centralized place for this doesn't sound too shabby.